2025-05-11 17:04:21,920 INFO: Dados recebidos: ImmutableMultiDict([('csrf_token', 'simulated-csrf-token-s2zi22m490n'), ('nome', 'Juninho Boni'), ('telefone', '(11) 11111-1111'), ('email', 'juninhoboni@gmail.com'), ('rua', 'Teste'), ('numero', '123'), ('complemento', ''), ('bairro', 'Teste'), ('cep', '14820-480'), ('cidade', 'Americo'), ('uf', 'AL'), ('produto', 'caderno'), ('tipo_caneca', 'simples'), ('cor_caneca', 'amarela'), ('tipo_caderno', 'brochura'), ('quantidade_de_paginas', '50'), ('quantidade', '1'), ('estampa', 'Estampa Minions'), ('obs', '')]) [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\app.py:177]
2025-05-11 17:04:22,078 INFO: Validando e-mail: juninhoboni@gmail.com [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\models.py:51]
2025-05-11 17:04:22,906 INFO: E-mail validado: juninhoboni@gmail.com [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\models.py:54]
2025-05-11 17:05:54,525 INFO: Aplicação de orçamentos iniciada [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\app.py:75]
2025-05-11 17:06:01,123 INFO: Dados recebidos: ImmutableMultiDict([('csrf_token', 'simulated-csrf-token-s2zi22m490n'), ('nome', 'Juninho Boni'), ('telefone', '(11) 11111-1111'), ('email', 'juninhoboni@gmail.com'), ('rua', 'Teste'), ('numero', '123'), ('complemento', ''), ('bairro', 'Teste'), ('cep', '14820-480'), ('cidade', 'Americo'), ('uf', 'AL'), ('produto', 'caderno'), ('tipo_caneca', 'simples'), ('cor_caneca', 'amarela'), ('tipo_caderno', 'brochura'), ('quantidade_de_paginas', '50'), ('quantidade', '1'), ('estampa', 'Estampa Minions'), ('obs', '')]) [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\app.py:177]
2025-05-11 17:06:01,252 INFO: Validando e-mail: juninhoboni@gmail.com [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\models.py:51]
2025-05-11 17:06:02,123 INFO: E-mail validado: juninhoboni@gmail.com [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\models.py:54]
2025-05-11 17:06:15,115 ERROR: Problema de codificação ao enviar para ['juninhoboni@gmail.com']. Certifique-se que todos os textos estão em UTF-8. Erro: 'ascii' codec can't encode character '\xe7' in position 731: ordinal not in range(128) [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\app.py:151]
Traceback (most recent call last):
  File "C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\app.py", line 122, in enviar_email
    yag.send(
  File "C:\Users\User\PycharmProjects\projetointegrador\.venv\Lib\site-packages\yagmail\sender.py", line 172, in send
    return self._attempt_send(recipients, msg_strings)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\PycharmProjects\projetointegrador\.venv\Lib\site-packages\yagmail\sender.py", line 178, in _attempt_send
    result = self.smtp.sendmail(self.user, recipients, msg_strings)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\smtplib.py", line 864, in sendmail
    msg = _fix_eols(msg).encode('ascii')
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
UnicodeEncodeError: 'ascii' codec can't encode character '\xe7' in position 731: ordinal not in range(128)
2025-05-11 17:06:16,245 ERROR: Problema de codificação ao enviar para ['michelepersonalizados2025@gmail.com']. Certifique-se que todos os textos estão em UTF-8. Erro: 'ascii' codec can't encode character '\xe7' in position 732: ordinal not in range(128) [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\app.py:151]
Traceback (most recent call last):
  File "C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\app.py", line 122, in enviar_email
    yag.send(
  File "C:\Users\User\PycharmProjects\projetointegrador\.venv\Lib\site-packages\yagmail\sender.py", line 172, in send
    return self._attempt_send(recipients, msg_strings)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\PycharmProjects\projetointegrador\.venv\Lib\site-packages\yagmail\sender.py", line 178, in _attempt_send
    result = self.smtp.sendmail(self.user, recipients, msg_strings)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\smtplib.py", line 864, in sendmail
    msg = _fix_eols(msg).encode('ascii')
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
UnicodeEncodeError: 'ascii' codec can't encode character '\xe7' in position 732: ordinal not in range(128)
2025-05-11 17:06:16,482 WARNING: Orçamento 35 salvo, mas emails falharam parcialmente [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\app.py:252]
2025-05-11 17:06:59,456 INFO: Aplicação de orçamentos iniciada [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\app.py:75]
2025-05-11 17:07:04,179 INFO: Dados recebidos: ImmutableMultiDict([('csrf_token', 'simulated-csrf-token-s2zi22m490n'), ('nome', 'Juninho Boni'), ('telefone', '(11) 11111-1111'), ('email', 'juninhoboni@gmail.com'), ('rua', 'Teste'), ('numero', '123'), ('complemento', ''), ('bairro', 'Teste'), ('cep', '14820-480'), ('cidade', 'Americo'), ('uf', 'AC'), ('produto', 'caneca'), ('tipo_caneca', 'simples'), ('cor_caneca', 'amarela'), ('tipo_caderno', 'brochura'), ('quantidade_de_paginas', '50'), ('quantidade', '1'), ('estampa', 'Estampa Princesas Disney'), ('obs', '')]) [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\app.py:176]
2025-05-11 17:07:04,327 INFO: Validando e-mail: juninhoboni@gmail.com [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\models.py:51]
2025-05-11 17:07:05,430 INFO: E-mail validado: juninhoboni@gmail.com [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\models.py:54]
2025-05-11 17:07:11,133 ERROR: Problema de codificação ao enviar para ['juninhoboni@gmail.com']. Certifique-se que todos os textos estão em UTF-8. Erro: 'ascii' codec can't encode character '\xe7' in position 731: ordinal not in range(128) [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\app.py:150]
Traceback (most recent call last):
  File "C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\app.py", line 121, in enviar_email
    yag.send(
  File "C:\Users\User\PycharmProjects\projetointegrador\.venv\Lib\site-packages\yagmail\sender.py", line 172, in send
    return self._attempt_send(recipients, msg_strings)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\PycharmProjects\projetointegrador\.venv\Lib\site-packages\yagmail\sender.py", line 178, in _attempt_send
    result = self.smtp.sendmail(self.user, recipients, msg_strings)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\smtplib.py", line 864, in sendmail
    msg = _fix_eols(msg).encode('ascii')
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
UnicodeEncodeError: 'ascii' codec can't encode character '\xe7' in position 731: ordinal not in range(128)
2025-05-11 17:07:12,968 ERROR: Problema de codificação ao enviar para ['michelepersonalizados2025@gmail.com']. Certifique-se que todos os textos estão em UTF-8. Erro: 'ascii' codec can't encode character '\xe7' in position 732: ordinal not in range(128) [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\app.py:150]
Traceback (most recent call last):
  File "C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\app.py", line 121, in enviar_email
    yag.send(
  File "C:\Users\User\PycharmProjects\projetointegrador\.venv\Lib\site-packages\yagmail\sender.py", line 172, in send
    return self._attempt_send(recipients, msg_strings)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\PycharmProjects\projetointegrador\.venv\Lib\site-packages\yagmail\sender.py", line 178, in _attempt_send
    result = self.smtp.sendmail(self.user, recipients, msg_strings)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\smtplib.py", line 864, in sendmail
    msg = _fix_eols(msg).encode('ascii')
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
UnicodeEncodeError: 'ascii' codec can't encode character '\xe7' in position 732: ordinal not in range(128)
2025-05-11 17:07:13,206 WARNING: Orçamento 36 salvo, mas emails falharam parcialmente [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\app.py:251]
2025-05-11 17:09:52,331 INFO: Aplicação de orçamentos iniciada [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\app.py:75]
2025-05-11 17:10:38,483 INFO: Dados recebidos: ImmutableMultiDict([('csrf_token', 'simulated-csrf-token-s2zi22m490n'), ('nome', 'Juninho Boni'), ('telefone', '(11) 11111-1111'), ('email', 'juninhoboni@gmail.com'), ('rua', 'Teste'), ('numero', '123'), ('complemento', ''), ('bairro', 'Teste'), ('cep', '14820-480'), ('cidade', 'Americo'), ('uf', 'AM'), ('produto', 'caneca'), ('tipo_caneca', 'simples'), ('cor_caneca', 'amarela'), ('tipo_caderno', 'brochura'), ('quantidade_de_paginas', '50'), ('quantidade', '1'), ('estampa', 'Estampa Stitch'), ('obs', '')]) [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\app.py:185]
2025-05-11 17:10:38,632 INFO: Validando e-mail: juninhoboni@gmail.com [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\models.py:51]
2025-05-11 17:10:41,048 INFO: E-mail validado: juninhoboni@gmail.com [in C:\Sites\Micheli-Personalizados-main\Micheli-Personalizados-main\models.py:54]
